test/integration/forms/form.js

```diff
@@ -1313,6 +1314,18 @@ context('Patient Action Form', function() {
       .find('.js-save-button')
       .click();
 
+    cy
+      .get('.fill-window--dark.is-shown')
+      .should('contain', 'Saving your work...');
+
+    cy
+      .get('.app-frame__content')
+      .click('left', { force: true });
+
+    cy
+      .get('.fill-window--dark.is-shown')
+      .should('exist');
+
```

Comment:
This makes sure the loading modal is shown after the `Save + Go Back` button is clicked. And that it doesn't close when the user clicks elsewhere on the page.